Fix flaky APIKeyTest test_invalidate_by_id

Looks like there is a new flaky test showing up:

**Build and Test / System and Environment Tests / test_apikey.APIKeyTest.test_invalidate_by_id** (from nosetests)

**Error Message**
```
{u'invalidated_api_keys': [], u'error_count': 0}
```

**Stacktrace**
```
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/case.py", line 329, in run
    testMethod()
  File "/go/src/github.com/elastic/apm-server/tests/system/test_apikey.py", line 90, in test_invalidate_by_id
    assert invalidated.get("invalidated_api_keys") == [apikey["id"]], invalidated
AssertionError: {u'invalidated_api_keys': [], u'error_count': 0}
```

Another test was failing where an API Key that should have been invalidated, wasn't. This might be related:

```
[2020-02-20T18:59:28.373Z] ======================================================================

[2020-02-20T18:59:28.373Z] FAIL: test_info_by_name (test_apikey.APIKeyTest)

[2020-02-20T18:59:28.373Z] ----------------------------------------------------------------------

[2020-02-20T18:59:28.373Z] Traceback (most recent call last):

[2020-02-20T18:59:28.373Z]   File "/go/src/github.com/elastic/apm-server/tests/system/test_apikey.py", line 122, in test_info_by_name

[2020-02-20T18:59:28.373Z]     assert len(info.get("api_keys")) == 2, info

[2020-02-20T18:59:28.373Z] AssertionError: {u'api_keys': [{u'username': u'apm_server_user', u'creation': 1582224716534, u'id': u'-WDxY3ABE8Fs4UHRM0IL', u'invalidated': False, u'name': u'apm_integration_key'}, {u'username': u'apm_server_user', u'creation': 1582224717693, u'id': u'-mDxY3ABE8Fs4UHRN0Ke', u'invalidated': False, u'name': u'apm_integration_key'}, {u'username': u'apm_server_user', u'creation': 1582224718607, u'id': u'-2DxY3ABE8Fs4UHRO0Iu', u'invalidated': False, u'name': u'apm_integration_key'}]}

```

--- a/tests/system/test_apikey_cmd.py
+++ b/tests/system/test_apikey_cmd.py
@@ -111,7 +111,7 @@ class APIKeyCommandBaseTest(BaseTest):
         return apikey
 
     def invalidate_by_id(self, id):
-        invalidated = self.subcommand_output("invalidate", "--id", id)
+        invalidated = self.subcommand_output("invalidate", "--id={}".format(id))
         self.apikey_helper.wait_until_invalidated(id=id)
         return invalidated
 
@@ -167,7 +167,7 @@ class APIKeyCommandTest(APIKeyCommandBaseTest):
     def test_info_by_id(self):
         self.create()
         apikey = self.create()
-        info = self.subcommand_output("info", "--id", apikey["id"])
+        info = self.subcommand_output("info",  "--id={}".format(apikey["id"]))
         assert len(info.get("api_keys")) == 1, info
         assert info["api_keys"][0].get("username") == self.user, info
         assert info["api_keys"][0].get("id") == apikey["id"], info
@@ -190,25 +190,26 @@ class APIKeyCommandTest(APIKeyCommandBaseTest):
 
     def test_verify_all(self):
         apikey = self.create()
-        result = self.subcommand_output("verify", "--credentials", apikey["credentials"])
+        result = self.subcommand_output("verify", "--credentials={}".format(apikey["credentials"]))
         assert result == {'event:write': True, 'config_agent:read': True, 'sourcemap:write': True}, result
 
         for privilege in ["ingest", "sourcemap", "agent-config"]:
-            result = self.subcommand_output("verify", "--credentials", apikey["credentials"], "--" + privilege)
+            result = self.subcommand_output(
+                "verify", "--credentials={}".format(apikey["credentials"]), "--" + privilege)
             assert len(result) == 1, result
             assert list(result.values())[0] is True
 
     def test_verify_each(self):
         apikey = self.create("--ingest")
-        result = self.subcommand_output("verify", "--credentials", apikey["credentials"])
+        result = self.subcommand_output("verify", "--credentials={}".format(apikey["credentials"]))
         assert result == {'event:write': True, 'config_agent:read': False, 'sourcemap:write': False}, result
 
         apikey = self.create("--sourcemap")
-        result = self.subcommand_output("verify", "--credentials", apikey["credentials"])
+        result = self.subcommand_output("verify", "--credentials={}".format(apikey["credentials"]))
         assert result == {'event:write': False, 'config_agent:read': False, 'sourcemap:write': True}, result
 
         apikey = self.create("--agent-config")
-        result = self.subcommand_output("verify", "--credentials", apikey["credentials"])
+        result = self.subcommand_output("verify", "--credentials={}".format(apikey["credentials"]))
         assert result == {'event:write': False, 'config_agent:read': True, 'sourcemap:write': False}, result
 
 

